.github/workflows/reusable-crypto3-publish-result.yml

```diff
@@ -4,11 +4,17 @@ name: Publish Crypto3 Test Results (Reusable)
 
 on:
   workflow_call:
+    inputs:
+      artifact-name:
+        type: string
+        description: "Name of the artifact to be published."
+        required: false
+        default: 'test-results'
```

Comment:
There is no need to pass specific artifact name. If it would be the same for both testing workflows, things gonna work anyway, because workflow_call callback is run for each testing workflow independently.
The detail I was referencing to, is that publish action will overwrite test results of one testing with another. It happens due to similar check names. We should set them to differently for each callback. 
They are set with `check_name` option passed to [publish action](https://github.com/EnricoMi/publish-unit-test-result-action#configuration)

Reply:
Ok, I misunderstood you. Reverted that change, made the correct change.